oughta.cabal

```diff
@@ -158,7 +158,7 @@ test-suite oughta-tests
   hs-source-dirs: test
   main-is: Test.hs
   build-depends:
-    oughta > 0.2.0.0,
+    oughta > 0.3.0.0,
```

Comment:
We should remove these version bounds at some point, as there's no point in having them: `cabal` will _always_ pick the in-tree version of `oughta` anyway. (I think `cabal check` might even warn about this under certain circumstances.)

Reply:
I'm not sure why I thought that this was necessary at some point... Will do a bit of digging